include/crypto/evp.h

```diff
@@ -746,6 +750,23 @@ struct evp_pkey_st {
 # define EVP_PKEY_CTX_IS_KEM_OP(ctx) \
     (((ctx)->operation & EVP_PKEY_OP_TYPE_KEM) != 0)
 
+struct evp_skey_st {
+    /* == Common attributes == */
+    CRYPTO_REF_COUNT references;
+    CRYPTO_RWLOCK *lock;
+
+    /*
+     * Special case - raw bytes key.
+     * When it is set, we MUST have keymgmt == NULL.
+     * We use this raw buffer the same way we used it before.
+     */
+    unsigned char *key;
+    size_t keylen;
```

Comment:
Might I suggest `keybytes` and `keybyteslen`?  Just `key` is a bit vague.

Reply:
Sure, done